test/e2e/argo_server_test.go

```diff
@@ -82,62 +80,49 @@ func (s *ArgoServerSuite) TestUnauthorized() {
 }
 
 func (s *ArgoServerSuite) TestPermission() {
-	s.T().SkipNow() // TODO
-	nsName := fmt.Sprintf("%s-%d", "test-rbac", time.Now().Unix())
-	ns := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: nsName}}
-	s.Run("Create ns", func(t *testing.T) {
-		_, err := s.KubeClient.CoreV1().Namespaces().Create(ns)
+	nsName := "argo"
+	// Create good serviceaccount
+	goodSaName := "argotestgood"
+	goodSa := &corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{Name: goodSaName}}
+	s.Run("Create good service account", func(t *testing.T) {
```

Comment:
I think test name should be SnakeCase?

Reply:
I don't see Snake Case in the file but Camel Case, though it doesn't require the testing name to be a "word". @alexec

Reply:
Changed to Camel Case.